CHANGES.md

```diff
@@ -1,3 +1,48 @@
+# 0.24.0
+
+New features:
+* CS2 rachel web serving benchmark (GH-451, GH-605)
+* Add [AliCloud](http://intl.aliyun.com/) provider (GH-611)
+* Add tomcat/wrk benchmark (GH-598)
+
+Breaking changes:
+* Remove `--parallelism` flag (GH-633)
+* Rename `fio` flags (GH-518, GH-581)
+* (Future breaking change) Rename disk types and disk-related flags (GH-599)
```

Comment:
Might want to change this to "Added new disk types and options to clarify use."
(breaking change) we will remove the current options in a coming release.